app/instructionalSupport/studentSupportCallForm/controllers/studentSupportCallFormCtrl.js

```diff
@@ -172,6 +172,17 @@ instructionalSupportApp.controller('StudentSupportCallFormCtrl', ['$scope', '$ro
 					return false;
 				}
 
+				if ($scope.view.state.supportCallResponse.requirePreferenceComments == true) {
+					for (var i = 0; i < $scope.view.state.preferences.length; i++) {
```

Comment:
No ES15 loop?

Reply:
I do agree with avoiding the basic for loop when possible, but in this case I want to prematurely terminate the loop based on a condition.